src/controller/genericmidi.cxx

```diff
@@ -623,28 +398,30 @@ void GenericMIDI::launchScene( int scene )
     
     if ( b->action == GRID_LAUNCH_SCENE )
     {
-      for( int i = 0; i < 5; i++ )
```

Comment:
New branch created "gmlaunch", for GenericMidi Launch, with one commit:
https://github.com/openAVproductions/openAV-Luppp/commit/b6175368e175147f4580ed75aeca51f1233e7988.